CONTRIBUTING.md

```diff
@@ -99,9 +99,29 @@ You'll probably also want to add your non-localhost domains to the `allowedHosts
 
 ## Finding something to work on
 
-If this is your first time here, check out the label [Good First Issue](https://github.com/firefox-devtools/profiler/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). We will mentor you through the process of completing a first bug, and these are usually pretty good self-contained problems. After leveling up on a few good first issues, we also have the [Polish](https://github.com/firefox-devtools/profiler/issues?q=is%3Aopen+is%3Aissue+label%3Apolish) tag for bugs that no one is actively working on, but are well-scoped and ready to be tackled!
-
-Make sure and comment on the issue letting someone know you are interested in working on an issue. Feel free to [chat with us on slack](https://devtools-html-slack.herokuapp.com/) if you need help finding something you might be interested to work on.
+If this is your first time here, check out the label
+[good first issue](https://github.com/firefox-devtools/profiler/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+-label%3Aassigned).
+We will mentor you through the process of completing a first bug, and these are
+usually pretty good self-contained problems.
+
+After leveling up on a few good first issues, we also have the
+[help wanted](https://github.com/firefox-devtools/profiler/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+-label%3A%22good+first+issue%22+-label%3Aassigned+)
+tag for bugs that no one is actively working on, but are well-scoped and ready
+to be tackled!
+
+We know how stimulating working on our project can be, but
+please refrain claiming several issues at once. Instead please decide on the one
+issue you're interested in and leave a comment to let someone know. Take
+especially care that no other contributor already asked for it. Then someone from
+the team will answer and assign the issue to you. If you're commenting during
+week-ends be aware that our team is working mostly on week days,
+therefore please be patient :-)
+
+
+Feel free to [chat with us on
+slack](https://devtools-html-slack.herokuapp.com/) if you need help finding
+something you might be interested to work on or have any question. You can also
+leave message on our special issue #1785.
```

Comment:
It looks like GitHub doesn't convert `#1785` to a link when it's in an md file: https://github.com/firefox-devtools/profiler/blob/f76e6e833e26ba53dfd4d3e0d6eacc354de87025/CONTRIBUTING.md#finding-something-to-work-on
Could you also make it a link?

Reply:
ah, thanks for catching it!